mozci/data/sources/artifact/__init__.py

```diff
@@ -61,9 +62,10 @@ def _load_errorsummary(self, task_id) -> None:
 
         missing_groups = groups - set(group_results)
         if len(missing_groups) > 0:
-            logger.error(
-                f"Some groups in {task_id} are missing results: {missing_groups}"
-            )
+            if not self.is_try:
+                logger.error(
+                    f"Some groups in {task_id} are missing results: {missing_groups}"
+                )
```

Comment:
We should continue to at least log this at the `debug` level for try pushes. According to [the loguru docs](https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.log) you should be able to do something like `logger.log(LEVEL, msg)` (where LEVEL is "DEBUG" or "WARNING").